wcf.targets

```diff
@@ -47,4 +47,9 @@
      <WcfToolsOutputPath>$(WcfRootFolder)bin\Wcf\tools\</WcfToolsOutputPath>
   </PropertyGroup>
 
+  <!-- Override ValidateIgnoreReference from Target 'ValidatePackageTargetFramework' in PackageLibs.targets -->
+  <ItemGroup>
+    <ValidateIgnoreReference Include="System.Private.ServiceModel" />
+  </ItemGroup>
+
```

Comment:
That's happening because the facade projects aren't referencing these libraries, they aren't coming as part of the project reference, and NuGet isn't flowing them because the x-plat NuGet doesn't yet call into msbuild to determn

You could either directly reference all these packages or disable S.P.SM *only* in the facade projects.  Plese don't disable globally or you'll lose all of your validation.